### ISSUE TITLE
draw.gam() should set widths = 1

### ISSUE THREAD

**Issue by gavinsimpson** [gavinsimpson/gratia#190]
The default behaviour in patchwork when aligning patches of plots where one or more has a fixed/equal coordinate system is not desirable for `draw.gam()` and related plotting functions as it reduces the widths of the columns to align the axes. In these situations we don't want to align the axes as we are specifically trying to preserve the aspect ratio for select plots.

The fix in this situation seems to be to set the `widths` argument such that all patches are the same size. See https://patchwork.data-imaginist.com/articles/guides/layout.html#fixed-aspect-plots

**Comment by Skogur** [comment]
I'm not sure if this is related or not, but will share in case it is helpful...

I have consistently been having issues when plotting 2D smooths via gratia::draw(). I was (& remain) unsure of if this was on `gratia`'s end or mine (perhaps multiple ggplot-related packages not playing nicely with one another?).  However, after wrestling with this issue for some time, I recently discovered that simply adding + theme(aspect.ratio = 1) fixed the problem. Graphical output examples below.

Figured i should share in case it were related here or others are seeking solution for this as well....

**### Before...**
```
  draw(gam1, select = smooths(gam1)[2] ) 
```
![image](https://user-images.githubusercontent.com/21228500/197832943-cda63871-ef30-4e37-9fc7-681634159b4b.png)


**### After...**
```
  draw(gam1, select = smooths(gam1)[2] ) + 
      theme(aspect.ratio = 1)
```
![image](https://user-images.githubusercontent.com/21228500/197832107-41c0dc86-769d-4a9d-ad08-039970cd42b3.png)

PS major thanks for all that you do!
